README.md

```diff
@@ -2,15 +2,54 @@
 
 ## Description
 
-Jiminy is an open-source C++ simulator of poly-articulated systems, under the first restriction that the contact with the ground can be reduced to a dynamic set of points and the second restriction that the collisions between bodies or the environment can be neglected.
+Jiminy is a fast and lightweight open-source simulator for poly-articulated systems. It was built with two ideas in mind:
 
-It is built upon [Pinocchio](https://github.com/stack-of-tasks/pinocchio), which is an open-source implementing highly efficient Rigid Body Algorithms for poly-articulated systems. It is used to handle low-level physics calculations related to the system, while the effect of the environment on it is handled by Jiminy itself. The integration of time is based on the open-source library [Boost Odeint](https://github.com/boostorg/odeint).
+ - **provide a fast yet physically accurate simulator for robotics research.** Jiminy is built around
+ [Pinocchio](https://github.com/stack-of-tasks/pinocchio), an open-source fast and efficient kinematics and
+ dynamics library. Jiminy thus uses minimal coordinates and Lagrangian dynamics to simulate an articulated
+ system: this makes Jiminy as close as numerically possible to an analytical solution, without the risk of
+ joint violation. Ground reaction is modeled as a spring-damper external force applied to the system.
```

Comment:
["Ground reaction is modeled as a spring-damper external force applied to the system"] for arbitrary collision geometry.